VERSION

```diff
@@ -1 +1 @@
-2.0.0
+6.5.2+5.da93ca823
```

Comment:
Why is this file needed? Where is it used? and why does it have the node version instead of the aerepl version?

Reply:
I think you can remove it

Reply:
Removed here 550c751bf4be84cb762c47299f174840a0c1299c